lib/absinthe/subscription.ex

```diff
@@ -118,13 +119,14 @@ defmodule Absinthe.Subscription do
       }
     }
 
-    {:ok, _} = Registry.register(registry, field_key, doc_value)
-    {:ok, _} = Registry.register(registry, {self(), doc_id}, field_key)
+    _ = Registry.register(uniq_registry, {self(), :context}, doc.execution.context)
+    {:ok, _} = Registry.register(dup_registry, field_key, doc_value)
+    {:ok, _} = Registry.register(dup_registry, {self(), doc_id}, field_key)
```

Comment:
Part of the reasons we have all of these keys is that we basically are using it to stash Absinthe specific meta-data about a given pid and its subscriptions somewhere. We _could_ use the process dictionary, since the whole point is that it is process specific state. Feels dirty though.